### PR TITLE
Formaction prototype

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: samples/RazorPages.Samples.Web/Pages/Index.razor [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿@page "{deleteId?}"
+﻿@page "{formaction?}/{id?}"
```
Comment (rynowak) [inline samples/RazorPages.Samples.Web/Pages/Index.razor]: @DamianEdwards - what do you think of this approach for formaction?

File: samples/RazorPages.Samples.Web/Pages/Index.razor [hunk 2]
```diff
@@ -24,7 +24,7 @@
                     <td>@Html.DisplayFor(_ => customer.Id)</td>
                     <td>@Html.DisplayFor(_ => customer.Name)</td>
                     <td><a class="btn btn-success" href="/Customer/Edit/@customer.Id">Edit</a></td>
-                    <td><button type="submit" formaction="@customer.Id" class="btn btn-danger">Delete</button></td>
+                    <td><button type="submit" formaction="delete/@customer.Id" class="btn btn-danger">Delete</button></td>
```
Comment (hishamco) [inline samples/RazorPages.Samples.Web/Pages/Index.razor]: What if we use tag helper for clarity like this
```asp-command="delete" asp-command-args="@customer.Id"```

**Comment by hishamco** [comment]
It will be nice if we fix issue #32 to support **action** along side with ```{id?}``` and let the developer pass the route ```delete/{id}``` or ```edit/{id}``` in formaction attribute or user sort of tag helper like what I mentioned before. So the markup will be clear and simple with no extra stuff need to add in the underlying APIs

**Comment by hishamco** [comment]
@rynowak what's the difference between this PR and the one is there https://github.com/aspnet/RazorPages/commit/cf7e3657e4377c22367b45cf4b6f01a25b0ab807?

**Comment by rynowak** [comment]
Just the merge order is different. What I checked in to master is pretty much the same as this PR.

To answer your question about `action` - using the `action` route value is a minefield. That has a special intrinsic meaning in MVC.

**Pull request closed without merging**